spec/html/proofer/fixtures/images/replaceAbsUrlSrc.html

```diff
@@ -0,0 +1,2 @@
+
+<img alt="sure thing" src="http://baseurl.com/gpl.png"/> base url is relative to root
```

Comment:
Please replace `http://baseurl.com` with `http://example.com` everywhere. You can omit the comment “ base url is relative to root” too.